data-contract/src/contract/invalidate.rs

```diff
@@ -105,11 +107,15 @@ impl CertificationContract {
         // Remove from owners map
         self.tokens.owner_by_id.remove(&token_id);
 
+        let empty_account_id: AccountId = unsafe {
+            std::mem::transmute(UnsafeAccountId(String::new()))
+        };
+
         // Emit nonstandard NFT transfer event
         NftTransfer {
             old_owner_id: &owner_id,
             // Intentionally invalid `new_owner_id` because a standard "NFT Deleted" event doesn't exist
-            new_owner_id: &AccountId::new_unchecked(String::new()),
```

Comment:
This causes tests to fail, since `new_unchecked` contains a debug-time assertion that still validates the input